### PR TITLE
BugFix: websocketserver not opening

### PR DESCRIPTION
**Pull request by lukasrgr**
- added missing variable due to websocketserverport not opening
- added over the air flashing

### CODE REVIEW

File: esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+/***************************** Struct definitions *****************************/
+/**************************** Prototype functions *****************************/
+/**************************** Variable definitions ****************************/
+AsyncWebServer asyncServer(8000);
+AsyncWebSocket ws("/ws");
+
+/********************************* Constants **********************************/
+const char index_html[] PROGMEM = R"rawliteral(
+<!DOCTYPE HTML><html>
```
Comment (timbleman) [inline esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp:17]: This HTML page could be moved in a separate file and imported, but this is fine too.

File: esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+</html>)rawliteral";
+
+/**************************** Function definitions ****************************/
+/**
+ * @brief Reacts to an event
+ *
+ * @param server the websocketServer
+ * @param client the websocketClient
+ * @param type the eventType
+ * @param arg
```
Comment (timbleman) [inline esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp:147]: Not too descriptive :D

**Review by timbleman — APPROVED** [review]
Approved. Thanks for implementing!

File: esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp (hunk 2 above)
Comment (lukasrgr) [reply esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp:147]: thanks

File: esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp (hunk 1 above)
Comment (lukasrgr) [reply esp8266_code/actual_project/src/hardware_specific_code/web_server/ota/ota.cpp:17]: could be

**Pull request merged**